tests/pytest_tests/system_tests/test_launch/test_job_status_tracker.py

```diff
@@ -41,6 +41,10 @@ async def test_check_stop_run_exist_stopped(user, wandb_init):
     mock_launch_project.target_entity = run._entity
     mock_launch_project.target_project = run._project
     mock_launch_project.run_id = run._run_id
+
+    api_run = api.run_config(project=run._project, entity=run._entity, run=run._run_id)
```

Comment:
How does this fix the flake?

Reply:
![image](https://github.com/wandb/wandb/assets/19414170/1f114edb-f3bc-4b85-8b57-673db9d54a13)